tools/pipelines/templates/build-docker-service.yml

```
@@ -234,7 +234,7 @@ stages:
           command: build
           dockerFile: ${{ parameters.buildDirectory }}/Dockerfile
           buildContext: ${{ parameters.buildDirectory }}
-          arguments: --target base
+          arguments: --target base --build-arg REGISTRY=mcr.microsoft.com/mirror/docker/library
```

Per <https://github.com/microsoft/FluidFramework/pull/10560/files#r891593068> I think you want to add `--pull`:

```suggestion
          arguments: --pull --target base --build-arg REGISTRY=mcr.microsoft.com/mirror/docker/library
```